### DISCUSSION TITLE
Loading more than one video into source

### DISCUSSION THREAD

**Discussion by StackZs** [TheWidlarzGroup/react-native-video#4335] [Ideas]
### What problem are you trying to solve?

I would like to use the native UI AV player on tvOS to switch between videos. I saw that a way to transfer more than one video to the source param has been considered for a long time.

### Proposed solution

Support loading more than one video into the `source` param, so that multiple videos can be passed and switched between using the native AV player UI on tvOS.

### Platforms

Apple tvOS

### Additional context

The user asks whether this is already being considered or planned for the future, since it has been considered for a long time.

**Comment by freeboub** [comment]
It needs to be implemented... You can check with @moskalakamil for commercial support I think

**Comment by TWG-AI** [comment]
Thanks for the request! Native playlist / multi-source support using AVQueuePlayer (iOS/tvOS) and ExoPlayer's media queue (Android) is a solid feature for podcast and video-queue apps. We've noted this for the team.
